bff-node/src/controllers/helloWorld.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
```

Comment:
Nice to have a dummy controller to be able to build the project structure. 👏 We should remember to delete this when we got "real" controllers.